internal/controller/serverclaim_controller.go

```diff
@@ -151,7 +153,7 @@ func (r *ServerClaimReconciler) reconcile(ctx context.Context, log logr.Logger,
 	}
 	if server == nil {
 		log.V(1).Info("No server found for claim")
-		return ctrl.Result{}, nil
+		return ctrl.Result{RequeueAfter: r.ResyncInterval}, nil
```

Comment:
I am not sure we should do a polling here. My idea was to improve the watch setup and requeue all `ServerClaims` if e.g. `metadata.labels` on a `Server` changes.

Reply:
that's a good point. We already watch on `Servers`, so we could just add a condition to reconcile unbound `ServerClaims` if any `Server` is labeled. I have added this bit and removed this requeue.